docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -4,5 +4,63 @@
 Vitamin A Deficiency
 ====================
 
+Disease Description
+-------------------
+
+Vitamin A deficiency (VAD) is a lack of vitamin A in blood and tissues.
+Vitamin A deficiency is considered as one of the most serious public health concerns in developing countries
+and can contribute directly or indirectly to disability. 
+
+GBD 2017 Modeling Strategy
+------------------------------------
+In Global Burden of Disease (GBD) 2017, VAD exposure definition is proportion of the population with serum retinol concentration <0·7 μmol/L.
+Like iron deficiency, the cause VAD is also a population attributable fraction (PAF) of 1 cause with the VAD risk factor. That is, 100% of the VAD cases are attributable 
+to the VAD risk factor. VAD Risk exposure and VAD cause prevalence data are the same.
```

Comment:
That's great to add the information about the exposure and cause prev. data being equal here. It will help someone in the future, who would have spent time investigating and coming to this conclusion!